sys/linux/socket_ip_tunnel.txt

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <uapi/linux/if_tunnel.h>
+include <uapi/linux/if.h>
+include <uapi/linux/in.h>
+include <uapi/linux/ip.h>
+include <net/ip.h>
+
+resource sock_ipv4[sock]
+resource sock_ipv6[sock]
+
+socket$ipv4_tunnel(domain const[AF_INET], type const[SOCK_DGRAM], proto const[0]) sock_ipv4
+socket$ipv6_tunnel(domain const[AF_INET6], type const[SOCK_DGRAM], proto const[0]) sock_ipv6
```

Comment:
These are identical to `socket$inet_udp` and `socket$inet6_udp` in socket_inet_udp.txt.  Is it possible to reuse those?

Reply:
Resolved by replacing them with `sock_udp` and `sock_udp6`.